core/embed/rust/src/ui/layout/obj.rs

```diff
@@ -20,13 +20,9 @@ use crate::{
     util,
 };
 
-#[cfg(any(feature = "model_t1", feature = "model_tr"))]
+#[cfg(not(feature = "model_tt"))]
```

Comment:
this was correct before

Buttons are enabled if we're building for `model_t1` or `model_tr`.

Touch is enabled if we're building for `model_tt`.

If we are building _both_ for `model_t1` and `model_tt`, both touch and buttons is enabled. There is no mutual exclusion. You can have buttons and touch in the same build.